free-3.md

```diff
@@ -1,43 +1,52 @@
-# Free project 3
+# Scheduling
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve users looking for a faster way to create and expand schedules. The DSL would "codify" scheduling by converting some text-based representation of the schedule into valid Google Calander or iCal format. This language might be a good way to meet the needs of people who enjoy the convenience of online schedules but find the process of filling it out and making changes or adding events very laborious. It might seem a bit silly at first to create an entire DSL for the sole purpose of saving a few minutes, but consider git. Github has both a terminal interface and a desktop alternative. Although some may consider the desktop interface to be limited in its functionality and less efficient than the terminal interface, others may find this more user-friendly and harder to mess up. Similarly, some may find terminal too hard to use or too easy to potentially mess up the repository. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea helps make scheduling easier by allowing for a text-based alternative. For instance, people who simply want to type up a few lines to generate a schedule for their semester classes rather than having to manually click and add each of their classes individually in the iCal interface. Rather than having to navigate through the calander to the specified month and date, potential users could simply write a one line command to schedule an event. Perhaps the biggest experience change with this DSL is that users would not have to actually open their calender app to input their schedule. As of now, users have to open their calender app and manually click through and create all their events.
 
```

Comment:
Very interesting idea for a DSL! I was skeptical until you mentioned git, but that's a really good point. I think that a visual interface would be more important in this scenario, especially if you wanted to see existing events in order to modify them. It would be interesting to figure out ways to make the events visible to the user both from the language interface as well as through an external application like Google Calendar.